src/Evolution/Systems/GeneralizedHarmonic/Equations.hpp

```diff
@@ -0,0 +1,64 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines class template GeneralizedHarmonicEquations.
+
+#pragma once
+
+#include "DataStructures/DataBox/Prefixes.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "DataStructures/Variables.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/System.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/TagsDeclarations.hpp"
+
+class DataVector;
+
+namespace GeneralizedHarmonic {
+template <size_t Dim>
+struct ComputeDuDt {
+ public:
+  using return_tags = typelist<Tags::dt<SpacetimeMetric<Dim>>,
+                               Tags::dt<Pi<Dim>>, Tags::dt<Phi<Dim>>>;
+  using argument_tags = typelist<
+      SpacetimeMetric<Dim>, Pi<Dim>, Phi<Dim>,
+      Tags::d<SpacetimeMetric<Dim>, tmpl::size_t<Dim>, Frame::Inertial>,
+      Tags::d<Pi<Dim>, tmpl::size_t<Dim>, Frame::Inertial>,
+      Tags::d<Phi<Dim>, tmpl::size_t<Dim>, Frame::Inertial>, ConstraintGamma0,
+      ConstraintGamma1, ConstraintGamma2, GaugeH<Dim>,
+      SpacetimeDerivGaugeH<Dim>, Lapse<Dim>, Shift<Dim>,
+      UpperSpatialMetric<Dim>, UpperSpacetimeMetric<Dim>,
+      TraceSpacetimeChristoffelFirstKind<Dim>,
+      SpacetimeChristoffelFirstKind<Dim>, SpacetimeChristoffelSecondKind<Dim>,
+      SpacetimeNormalVector<Dim>, SpacetimeNormalOneForm<Dim>>;
+
+  /*!
+   * /brief Compute the RHS of the Generalized Harmonic formulation of
+   * Einstein's equations.
+   *
+   * /details For the full form of the equations see "A New Generalized Harmonic
+   * Evolution System" by Lindblom et. al, gr-qc/0512093.
```

Comment:
I would provide the full URL to the arxiv paper so that it can just be copy-pasted